types/tizen-common-web/index.d.ts

```diff
@@ -1,84 +1,14699 @@
+ *
+ * It represents the query statement for the specified value of _matchValue_ by the rule of _matchFlag_.
+ *
+ * If no _matchValue_ is defined, the filter matches all objects that have the attribute
+ * defined (same as the "EXISTS" filter works), otherwise, it only matches objects which have an attribute that match
+ * the specified value.
+ */
+export class AttributeFilter extends AbstractFilter {
+    constructor(attributeName: string, matchFlag?: FilterMatchFlag | null, matchValue?: any);
+    /**
+     * The name of the object attribute used for filtering.
+     *
+     * This is the name of the object attribute exactly as it is defined in
+     * the object's interface. For attributes of complex type, use fully-qualified names
+     * (such as "geolocation.latitude" to filter a video or image content's latitude in a geolocation).
+     *
+     * For attributes of an array type, the filter will match if any value in the array
+     * matches.
+     */
+    attributeName: string;
+    /**
+     * The match flag used for attribute-based filtering.
+     *
+     * By default, this attribute is set to "EXACTLY".
+     */
+    matchFlag: FilterMatchFlag;
+    /**
+     * The value used for matching.
+     *
+     * The filter will match if the attribute value matches the given matchValue.
+     *
+     * This value is not used if the _matchFlag_ is set to "EXISTS".
+     * By default, this attribute is set to null.
+     */
+    matchValue: any;
+}
+export interface AttributeFilter {
+    prototype: AttributeFilter;
+    new(attributeName: string, matchFlag?: FilterMatchFlag | null, matchValue?: any): AttributeFilter; // tslint:disable-line:no-misused-new
+}
+/**
+ * _AttributeRangeFilter_ represents a filter based on an object attribute
+ * which has values that are within a particular range.
+ *
+ * Range filters, where only one boundary is set, are available.
+ */
+export class AttributeRangeFilter extends AbstractFilter {
+    constructor(attributeName: string, initialValue?: any, endValue?: any);
+    /**
+     * The name of the object attribute used for filtering.
+     *
+     * The value of this attribute is exactly as it is defined in the object's interface. For attributes of complex type, use fully-qualified names
+     * (such as "geolocation.latitude" to filter a video or image content's latitude in a geolocation).
+     *
+     * For attributes of array type, the filter will match if any value in the array
+     * matches.
+     */
+    attributeName: string;
+    /**
+     * Objects with an attribute that is strictly lower than or equal to _endValue_ will match.
+     *
+     * By default, this attribute is set to null.
+     */
+    endValue: any;
+    /**
+     * Objects with an attribute that is greater than or equal to _initialValue_ will match.
+     *
+     * By default, this attribute is set to null.
+     */
+    initialValue: any;
+}
+export interface AttributeRangeFilter {
```

Comment:
Another case of incorrectly declaring the *static* (i.e., constructor) side of the class on an interface that represents the *instance*. Please remove.